70. Climbing Stairs.py

```diff
@@ -0,0 +1,22 @@
+class Solution:
+    def climbStairs(self, n):
+        """
+        :type n: int
+        :rtype: int
+        """
+        def climb(n, curr, memo):
+            count = 0
+            
+            if curr in memo:
+                return memo[curr]
+            if curr == n:
+                return 1
+            elif curr > n:
+                return 0
+            
+            count += climb(n, curr + 1, memo) + climb(n, curr + 2, memo)
```

Comment:
how come we need to do `count +=`? Can we not just do:
`memo[curr] = climb(n, curr + 1, memo) + climb(n, curr + 2, memo)`
`return memo[curr]`